publisher/monitor/manage-monitor/initialize_lightsail.sh

```diff
@@ -11,4 +11,4 @@ sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubun
 apt-cache policy docker-ce
 sudo apt install -y docker-ce
 sudo usermod -aG docker ${USER}
-(crontab -l 2>/dev/null; echo -e "0 * * * * docker build monitor/balance -t monitor-balance && docker run --rm --env-file monitor/balance/.env monitor-balance > monitor/log-balance.txt 2>&1 \n0 * * * * docker build monitor/price -t monitor-price && docker run --rm --env-file monitor/price/.env monitor-price > monitor/log-price.txt 2>&1") | crontab -
+(crontab -l 2>/dev/null; echo -e "0 * * * * docker build monitor/balance -t monitor-balance && docker run --rm --env-file monitor/balance/.env --env-file monitor/balance/.secrets.env monitor-balance > monitor/log-balance.txt 2>&1 \n0 * * * * docker build monitor/price -t monitor-price && docker run --rm --env-file monitor/price/.env --env-file monitor/price/.secrets.env monitor-price > monitor/log-price.txt 2>&1") | crontab -
```

Comment:
removing the secret from the docker build is a good pattern. I read more here: https://docs.docker.com/engine/swarm/secrets/

using env variables and remove seems roughly consistent with the recommendations here https://pythonspeed.com/articles/docker-build-secrets/

Reply:
👍  To be clear we never had secrets during docker build, before we just didn't need secrets at all and now we do.